### DISCUSSION TITLE
Why Datagrid example available only for angular ?

### DISCUSSION THREAD

**Discussion by ad9999** [vmware-clarity/core#148] [Q&A] [answered]
https://clarity.design/documentation/datagrid/structure

Component status page below shows its available for UI, NG.
https://clarity.design/documentation

**Comment by kevinbuhmann** [comment, marked as answer]
The only datagrid available in Clarity is in ng-clarity (@clr/angular). The Core datagrid (@cds packages) is neither complete nor supported.

You may be able to build a datagrid using only the @clr/ui styles, but I don't think that is really a workable solution given accessibility concerns. I will ask about changing that on the website.

**Reply by ad9999** [reply]
The table on [this](https://clarity.design/documentation) page says DataGrid is `Done`. This is not clear IMO. Developers sometimes not see the situation / documentation in same way as the dev / core team does. Some piece of documentation / example to support what you just said would be helpful.

**Reply by kevinbuhmann** [reply]
I think part of the confusion here might be due to the fact that https://clarity.design is now the ng-clarity website. The ng-clarity datagrid is "done" in the sense that it is shipped and available for use.

We switched the websites a few weeks ago. The Core documentation is now at https://core.clarity.design.

**Reply by ad9999** [reply]
Better have sub-domain like `ng-clarity.design` or `clarity.design/angular` so to avoid further confusion. The table in general as I mentioned earlier should be removed and placed under each project or should reflect the correct status. The `ng-clarity` and `core-` support for DataGrid are different

Links and proper call out need to happen at root level of project pages. 

Thanks for prompt response

**Reply by kevinbuhmann** [reply]
The table on https://clarity.design/documentation doesn't mention Core support because it's the ng-clarity website. It does have the correct status. We know the Clarity websites are confusing, but we aren't currently able to do the work needed to overhaul the documentation. Please bear with us as we are a small team.

**Reply by ad9999** [reply]
I mean `UI` column. What does that stand for on that page ? 

Is there any page for entire clarity framework showing component status between angular and core?

**Reply by ashleyryan** [reply]
UI is @clr/ui, which is the HTML/CSS implementation. No javascript, so most components aren't interactive, you just get styling. @clr/ng uses clr/ui to style its components.

No, there is no page comparing Core with Angular. Your best bet is to look at the two storybooks:

storybook.clarity.design
storybook.core.clarity.design

**Reply by chadgates** [reply]
Given that datagrid is missing in the https://core.clarity.design/get-started/roadmap/, does that mean, that there will not be a datagrid component any time soon in core ?

**Reply by ashleyryan** [reply]
That's correct, we have no immediate plans to release the core datagrid
